src/Core/src/AbstractApi.php

```diff
@@ -136,6 +136,15 @@ protected function getSignatureScopeName(): string
             ]
         );
 
+        if (\filter_var($configuration->get('debug'), \FILTER_VALIDATE_BOOLEAN)) {
+            $this->logger->debug('AsyncAws HTTP request sent', [
+                'method' => $request->getMethod(),
+                'endpoint' => $request->getEndpoint(),
+                'headers' => json_encode($request->getHeaders()),
+                'body' => 0 === $length ? null : $requestBody,
```

Comment:
is it an issue when body is a stream ?

Reply:
We've converted the body to a string just a few lines above.